src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -318,22 +328,25 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
-            if (commandResult.shouldDisplayTagsPanel()) {
-                handleViewTags(true);
-            }
-
             if (commandResult.shouldDisplayContactsPanel()) {
                 handleViewContacts(true);
             }
 
+            if (commandResult.shouldDisplayTagsPanel()) {
+                handleViewTags(true);
+            }
+
             if (commandResult.shouldDisplayEventsPanel()) {
                 handleViewEvents(true);
             }
 
-            if (!commandResult.shouldDisplayTagsPanel() && !commandResult.shouldDisplayContactsPanel()
-                    && !commandResult.shouldDisplayEventsPanel()) {
+            boolean shouldReturnToHome = commandResult.shouldReturnToHome();
```

Comment:
oh I thought it would have been better for code readability to store it in a variable and then pass it in to the `setReturnToHomeStatus()` rather than it being `setReturnToHomeStatus(commandResult.shouldReturnToHome())` but it always has been a personal preference of mine so idm changing

Reply:
In that case should it be a different name? Since it has the same name as the attribute that is in the `MainWindow` already